content/application-faq/application.md

```diff
@@ -43,9 +43,13 @@ A perfect application takes 30 days, if your supporting documents are accepted t
  around. If you are asked for tweaks to your supporting documents, expect 50-60 days.
 
 ## Do I need to list an address in Taiwan on the application?
-It’s entirely normal and accepted to apply with no local address. Once you get a place to stay,
+Yes. It’s entirely normal and accepted to apply with no local address. Once you get a place to stay,
```

Comment:
Take another read of the question :) It's not necessary to list an address, so this change is wrong.